en/docs/observability/root-cause-analysis.md

```diff
@@ -0,0 +1,99 @@
+# Root Cause Analysis
+
+Performing root cause analysis is crucial in identifying and rectifying the underlying problem for any anomalies detected in a system. Choreo Observability provides helpful insights out-of-the-box that assists in carrying out a root cause analysis for any service deployed on the Choreo platform.
+
+!!! info
+    For explanation, we are using the sample data available in the product.
+    
+## Get started with the sample service
+
+1. Go to **Services** from the left navigation.
+2. If this is your first service, then enter a name for the service and click **Create**. Skip step 3 and go to step 4.
+       ![Create a new service](/assets/img/observability/first-service-creation.png)
+3. Else, click **+ Create** to create a new service.
+       ![Create a new service](/assets/img/observability/service-creation.png)
+    1. Enter a name for the service and click **Create**.
+4. From the left-hand menu, click **Observe**.
+5. On the right-hand bottom, click the **Sample Service** icon.
+       ![Sample service popup](/assets/img/observability/sample-service-pop-up.png)
+6. Click **Try Sample Service**.
+
+       ![Try sample service](/assets/img/observability/try-sample-service.png)
+
+  
+    
+## Detecting anomalies and performing root cause analysis
+You can detect the anomalies of the usage of a service by observing the **Throughput & Latency** graphs. The throughput graph shows the successful and erroneous requests that occurred during a period. The latency graph shows the latency of each request. You can observe these graphs at a lower granularity by choosing a shorter time range, which gives you a better understanding of the incidents.
+
+### Analysing the throughput graph
+The throughput graph will show the throughput of requests per hour for a selected timestamp. 
+
+With the sample data, you will observe spikes in the throughput graph. Let's see how you can find
+the root cause by analyzing this data.
+
+Follow the steps below. 
+
+1. Hover over the throughput graph at 01/04/2021 13:41:00. You can observe that 46.06 errors had occurred at the time. 
+    
+    !!!tip
```

Comment:
Shall we use `>**Tip:**` ?

Reply:
Using this gives no color coding to the admonitions, right? For e.g. Info, warning, tip, error will all be displayed the same, isn't it? The inbuilt admonitions for the material theme, has this color theme.